src/realm/impl/sequential_getter.hpp

```diff
@@ -110,8 +111,8 @@ class SequentialGetter : public SequentialGetterBase {
             return global_end - m_leaf_start;
     }
 
-    size_t m_leaf_start;
-    size_t m_leaf_end;
+    size_t m_leaf_start = 0;
+    size_t m_leaf_end = 0;
```

Comment:
If we leave it uninitialised, then valgrind can tell us about the use of uninitialised values if we forget to initialise it which is useful in tracking down bugs. This might be a point of discussion comparing the value of these two tools. @rrrlasse  do you have a preference here?